src/components/table/renderer.tsx

```diff
@@ -18,7 +18,15 @@ export default class Table extends React.PureComponent<Props> {
 
     const tableBody = this.props.data.map((row, i) => {
       const rowNodes = this.props.header.map((field, j) => {
-        const { tooLong, formatted } = this.formatValue(row[field]);
+        let tooLong = false;
+        let formatted = '';
+        if (field !== 'month_date') {
```

Comment:
This is not a general fix. You don't know the field name for dates in general.

Reply:
Oh! okay , will fix it